main.go

```diff
@@ -886,8 +910,22 @@ func transformGo(file *ast.File, info *types.Info, blacklist map[types.Object]st
 			}
 			buildID = id
 		}
+
+		if token.IsExported(node.Name) || !envGarbleTiny {
```

Comment:
document why we're not doing the super-short names for exported names or private ones when -tiny isn't given